license.txt

```diff
@@ -174,3 +174,30 @@
       of your accepting any such warranty or additional liability.
 
    END OF TERMS AND CONDITIONS
+
+-----
+
+This project (specifically BaseKafkaSpec.scala) contains code derived from 
```

Comment:
Could we add the license to `BaseKafkaSpec.scala` as well?

Reply:
sure :+1:

Reply:
Done - by the way, should we add copyright/license headers to the test files?